README.md

```diff
@@ -124,10 +127,41 @@ Put the `PUSH_CREDENTIAL_SID` configuration info into your application server by
     PUSH_CREDENTIAL_SID = 'CR***'
     APP_SID = 'AP***'
 
-Once you’ve done that, restart the server so it uses the new configuration info. Now it's time to test. Hit your application server's `placeCall` endpoint. This will trigger a Twilio REST API request that will make an inbound call to your mobile app. Once your app accepts the call, you should hear a congratulatory message.
+Once you’ve done that, restart the server so it uses the new configuration info. Now it's time to test. Hit your application server's `placeCall` endpoint. This will trigger a Twilio REST API request that will make an inbound call to your mobile app. You will receive a incoming call notification.
 
 <img height="500px" src="images/quickstart/incoming_notification.png">"
 
+if your applicaion is in the foreground, you will see an alert dialog.
+
+<img height="500px" src="images/quickstart/incoming_call.png">"
+
+Once your app accepts the call, you should hear a congratulatory message.
+
+### <a name="bullet10"></a>10. Make client to client call
+
+To make client to client calls, follow step 6 through step 8 to run the application in two devices. Press the call button to connect to Twilio.  
```

Comment:
`to run the application on an additional device. Make sure to use a different identity in your access token.`

Why do they need to press the call button in this step?

Reply:
First call button press brings up the call dialog.